cmake_install.cmake

```diff
@@ -0,0 +1,49 @@
+# Install script for directory: /home/wizard/openSource/dpp_tests/cs_test/libs/DPP
```

Comment:
Please remove this file from the PR, this is specific to your computer, created when you did `cmake .`. In the future you should make a subdirectory, e.g. `mkdir build && cd build && cmake ..`.

Reply:
Opps sorry but the thing is the way i was doing it was testing it with a test script but i defo need to do this in the future